vhdbuilder/packer/install-dependencies.sh

```diff
@@ -504,6 +504,25 @@ EOF
 
 fi
 
+if grep -q "GB200" <<< "$FEATURE_FLAGS"; then
```

Comment:
is there a reason why this logic isn't baked in the new GB200 VHD only ?  only this "FEATURE_FLAGS" is used for that ?

Reply:
the string FEATURE_FLAG makes it non obvious that this is specific to VHD building.  we should look at renaming this to VHD_xxxxx